modules/ROOT/pages/telemetry-exporter.adoc

```diff
@@ -0,0 +1,146 @@
+= Export Anypoint Platform Data to Third-Party Apps
+
+The Telemetry Exporter feature enables you to export xref:access-management::audit-logging.adoc[_audit logs_] to third-party observability and analytics apps, such as Azure Monitor, Datadog, and Splunk HEC.
+//trace data should be a link once the topic exists
+
+To use the Telemetry Exporter feature, you must create a _connection_ with a third-party app to authorize Anypoint Platform to share data. You must then create a _configuration_ to specify which business groups allow content to be exported to connections. You can also edit and delete connections and configurations. 
```

Comment:
I suggest using bullets:
To use the Telemetry Exporter feature:
* Create a _connection_...
* Create a configuration_...

Reply:
They're being introduced in the context of the feature, which is why I did that, but I'll remove it.

Reply:
```suggestion
To use the Telemetry Exporter feature, you must: 

* Create a connection with a third-party app to authorize Anypoint Platform to share data
* Create a configuration to specify which business groups allow content to be exported to connections 

You can also edit and delete connections and configurations. 
```